src/Paket.Core/NuGetV2.fs

```diff
@@ -331,15 +330,8 @@ let ExtractPackage(fileName:string, targetFolder, name, version:SemVerInfo) =
         if  isExtracted fileName then
              verbosefn "%s %A already extracted" name version
         else
-            use zip = ZipFile.Read(fileName)
             Directory.CreateDirectory(targetFolder) |> ignore
-            for e in zip do
-                try
-                    e.Extract(targetFolder, ExtractExistingFileAction.OverwriteSilently)
-                with
-                | :? Ionic.Zip.BadCrcException as exn -> 
-                    traceWarnfn "Bad Crc during unzipping %s in %s %A: %s" e.FileName name version exn.Message 
-                | exn -> failwithf "Error during unzipping %s in %s %A: %s" e.FileName name version exn.Message 
+            ZipFile.ExtractToDirectory(fileName, targetFolder)
```

Comment:
Is overwrite silently the default behavior? And how about Crc exn? Not thrown anymore?